lib/bitboxnetwork.js

```diff
@@ -28,7 +28,7 @@ module.exports = class BitboxNetwork {
 
         for(let txOut of UTXO) {
             try {
-                txOut.slp = slpjs.decodeTxOut(txOut);
+                txOut.slp = slp.decodeTxOut(txOut);
```

Comment:
this is gonna look weird if youre just looking at the commit. further down `slp` was used a few times instead of `slpjs`, so this is changed to match.